gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/InstanceResource.java

```diff
@@ -79,4 +82,8 @@ public MonitoringResource getMonitoringResource() {
     private boolean isInstanceAccessibleByEnv(Set<String> environments, String currentEnvironment) {
         return environments == null || environments.isEmpty() || environments.contains(currentEnvironment);
     }
+
+    private Boolean cloudEnabled() {
+        return environment.getProperty("cloud.enabled", Boolean.class, false);
```

Comment:
Instead of using the Environment instance from Spring, you should use `parameterService.findAsBoolean`.
Check https://github.com/gravitee-io/gravitee-api-management/blob/99b62044fabeabbe43bb209764097a43305c9423/gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/organization/ConsoleSettingsResource.java#L109